vars/external_vars.yml.sample

```diff
@@ -13,9 +13,12 @@ INFLUXDB_PASSWORD: 'project'
 
 # Password for the grafana user who will connect to the grafana database
 INFLUXDB_GRAFANA_PASSWORD: 'grafana'
-#
+
 # Version of grafana you want to install
 GRAFANA_SOURCE_FILENAME: 'grafana-1.8.1'
 
+# Port for grafana nginx virtual server
+GRAFANA_NGINX_PORT: 8080
```

Comment:
@jwfang can you change your pull request so that the default value here is 80 and not 8080?